.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "cmsis-svd"]
-	path = cmsis-svd
-	url = git://github.com/posborne/cmsis-svd.git
```

Comment:
why was this submodule removed?

Reply:
Ups. Instead of drop, I squashed this commit.